src/psyclone/dynamo0p3.py

```diff
@@ -734,24 +726,24 @@ def function_space_to(self):
         ''' Return the "to" function space for a gh_operator. This is
         the first function space specified in the metadata. Raise an
         error if this is not an operator. '''
-        if self._type in VALID_OPERATOR_NAMES:
+        if self._type in GH_VALID_OPERATOR_NAMES:
```

Comment:
Not strictly modified by you but since you've altered the `if`, please could you fix pylint's complaint about the unnecessary `else` below.

Reply:
Done. I admit I noticed those warnings, but I usually prefer to have patches as 'clean' as possible, so that it is easy for the reviewer to see the important changes. I was thinking of a separate pull request that improves coding style ;) But, if you want it, you get it :)  Done.